linux/multipath.go

```diff
@@ -19,7 +19,7 @@ var (
 	showMapsFormat       = []string{"show", "maps", "format", "%w %d %n %s"}
 	orphanPathRegexp     = regexp.MustCompile(getOrphanPathsPattern())
 	multipathMutex       sync.Mutex
-	deviceVendorPatterns = []string{"Nimble", "3PARdata", "TrueNAS", "FreeNAS"}
+	DeviceVendorPatterns = []string{"Nimble", "3PARdata", "TrueNAS", "FreeNAS"}
```

Comment:
I made this global variable so that I can access this from a different file/package. Instead of maintaining two copies, I just made this a global variable so that any changes to this variable will need updating in a single place.